bool move_fwd
bool move_bkwd
